core/utils/templateMappings.ts

```diff
@@ -208,21 +183,13 @@ export const buildTemplate = async (appConfig: ConfigType): Promise<ConfigType>
   }
 
   if (!config.product) {
-    config.product = await ProductPrompt.askProduct();
-
-    if (config.product === 'universal') {
-      if (config.chain === 'flow') {
-        config.template = 'nextjs-flow-universal-wallet';
-      } else {
-        config.template = 'nextjs-universal-wallet';
-      }
-    } else if (config.chain === 'flow') {
+    config.product = 'dedicated';
```

Comment:
I think we can work on removing `product` from the config altogether @PBillingsby